queries/djot/highlights.scm

```diff
@@ -85,106 +90,163 @@
 ] @markup.list
 
 (list_marker_task
-  (unchecked) @constant.builtin) @markup.list.unchecked
+  (unchecked)) @markup.list.unchecked
```

Comment:
Are you marking the unchecked, or the marker, or is the marker has some other child nodes other than unchecked/checked?

Reply:
`unchecked` here is just marking a single space. So `- [ ]` is the `list_marker_task` and the space is the `unchecked` element. It didn't make much sense to mark the space, but I'm not sure what the best practice here is.

Reply:
I think the convention is to capture the whole `[  ]` and `[X]` (in case people want to conceal it).

Reply:
Oh okay, I guess this would need an update to the parser then.

Reply:
I think the idea is to replace `[x]`, not just the `x` (check markdown).

Reply:
yes.

Reply:
(mostly just the queries, not the parser). Haskell parser also have unicode characters as nodes, but using them in queries break it.

Reply:
That's different, and will be fixed by https://github.com/tree-sitter/tree-sitter/pull/3223.

Reply:
I've removed all the special characters from conceal.